docs/k0s-multi-node.md

```diff
@@ -29,17 +36,17 @@ To get a token run the following on one of the existing controller nodes:
 k0s token create --role=worker
 ```
 
-This will output a long [token](#tokens) which we will use to join the worker. To enhance security, we can also set an expiration time on the tokens by using:
+This will output a long [token](#tokens) which we will use to join a worker to the cluster. To enhance security, we can also set an expiration time for the token by using:
 ```sh
-k0s token create --role=worker --expiry="100h"
+$ k0s token create --role=worker --expiry=100h > token-file
 ```
 
 
-## Joining worker(s) to cluster
+## Adding Workers to a Cluster
 
-To join the worker we need to run k0s in worker mode with the token from previous step:
+To join the worker we need to run k0s in worker mode with the token from the previous step:
 ```sh
-$ k0s worker "long-join-token"
+$ k0s install worker --token-file /path/to/token/file
 ```
 
 That's it, really.
```

Comment:
We have this sentence already a few too many times in the documentation ;) Let's drop it here.